aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,320 @@
+
+    * **Container app name**: Leave the default value or enter a name of your choosing.
+    * **Subscription name**: Select the subscription you'd like to deploy to.
+    * **Resource group**: Select **New** and create a new resource group called *msdocs-scalable-razor*.
+    * **Container apps environment**: Select **New** to open the container apps environment dialog and enter the following values:
+        * **Environment name**: Leave the default value or enter a name of your choosing.
+        * **Location**: Select a location near you.
+        * **Azure Log Analytics Workspace**: Select **New** to open the log analytics workspace dialog.
+            * **Name**: Leave the default value or enter a name of your choosing.
+            * **Location**: Select a location near you and then choose **Ok** to close the dialog.
+        * Select **Ok** to close the container apps environment dialog.
+    * Select **Create** to close the original container apps dialog and Visual Studio creates the container app resource in Azure.
+1. Once the resource is created, make sure it's selected in the list of container apps, and then choose **Next**.
+1. You'll need to create an Azure Container Registry to store the published image artifact for your app. Select the green **+** icon on the container registry screen. Leave the default values, and then select **Create**.
+
+    :::image type="content" source="./media/scaling-new-registry-small.png" lightbox="./media/scaling-new-registry.png" alt-text="A screenshot showing how to create a new container registry.":::
+
+1. After the container registry is created, make sure it's selected, and then choose finish to close the dialog workflow and display a summary of the publishing profile.
+1. Select **Publish** in the upper right of the publishing profile summary to deploy your app to Azure.
+
+When the deployment finishes, Visual Studio launches the browser to display your hosted app. Search for `Microsoft` in the form field, and you should see a list of repositories displayed.
+
+## Scale and troubleshoot the app
+
+Your app is currently running without any issues, but you'd like to scale the app across more instances in anticipation of high traffic volumes.
+
+1. In the Azure portal, search for the `razorscaling-app-****` container app in the top level search bar and select it from the results.
+1. On the overview page, select **Scale** from the left navigation, and then select **+ Edit and deploy**.
+1. On the revisions page, switch to the **Scale** tab.
+1. Set both the min and max instances to **4** and then select **Create**. This configuration change guarantees your app is scaled horizontally across several instances.
+
+Navigate back to your app in the browser. When the page loads, at first it appears everything is working correctly. However, when you enter in a search term again and hit submit, an error occurs. If you don't see the error at first, submit the form several more times.
```

Comment:
```suggestion
Navigate back to the app. When the page loads, at first it appears everything is working correctly. However, when  a search term is entered and submitted, an error occurs. If an error is not displayed, submit the form several more times.
```
 Do we want to say. If the same instance responds to the form submission, no error occurs.